packages/ui-view/src/View/index.js

```diff
@@ -572,7 +616,7 @@ class View extends Component {
 View.omitViewProps = (props, Component) => {
   if (process.env.NODE_ENV !== 'production') {
     Object.keys(pickProps(props, View.propTypes)).forEach((prop) => {
-      error(false, `[${Component.displayName}] prop '${prop}' is not allowed.`)
+      error(false, `[${Component.name}] prop '${prop}' is not allowed.`)
```

Comment:
@matyasf this is the only real change in the file, the rest is made by prettier "lint on save"